### DISCUSSION TITLE
[Duet AI ] Colorize inline code suggestions

### DISCUSSION THREAD

**Discussion by ParensAllTheWayDown** [GoogleCloudPlatform/cloud-code-intellij#3188] [Ideas]
Please consider adding a color picker for the inline code suggestions.

The current behavior (at least in Pycharm) is using a light grey color that is very close to the comment color.  I find that I don't pick up on the suggestion because it is using a comment color.  
Suggest a color picker with the current default chosen, but the user can change the color for future responses.
I have used other products that do this and I find that it really helps me identify the suggestion and consider whether it is valid or not.

**Comment by ivanporty** [comment]
Thanks for the suggestion @ParensAllTheWayDown. We are looking into feasibility of this
